taar/hbase_client.py

```diff
@@ -38,11 +46,15 @@ def get_client_profile(self, client_id):
         """Retrieve the latest row for the given client in HBase
 
         Only the last known version of the info is retrieved"""
-        with contextlib.closing(Connection(self.hbase_hostname)) as connection:
-            table = connection.table(self.tablename)
-            row_start = "{}:{}".format(client_id, "99999999")
-            for key, data in table.scan(row_start=row_start, limit=1,
-                                        columns=[self.column_family],
-                                        reverse=True):
-                return json.loads(data[self.column].decode("utf-8"))
+        try:
+            with contextlib.closing(Connection(self.hbase_hostname)) as connection:
+                table = connection.table(self.tablename)
+                row_start = "{}:{}".format(client_id, "99999999")
+                for key, data in table.scan(row_start=row_start, limit=1,
+                                            columns=[self.column_family],
+                                            reverse=True):
+                    return json.loads(data[self.column].decode("utf-8"))
+        except Exception:
+            logger.exception("Connection to HBase failed for client {}".format(client_id))
```

Comment:
Let's use `extra` to store the client id and truncate the message after `failed`